source/overview.rst

```diff
@@ -1,5 +1,13 @@
 .. _overview:
 
+.. note::
+    Overall, with this section, I'm not quite sure why it's split into two
+    parts given there is so much overlap between the two of them. I also feel
+    that "Overview" is the wrong title, perhaps "Workflow" would be better.
+    I would really like an "Overview" section that gives me a more holistic
+    view of what benefits I would get by going through the (heavy) effort that
+    this section describes, but I haven't found it yet.
+
```

Comment:
I made some changes to this section, but feel free to submit a PR with more revisions.

Reply:
I think my thoughts here are broader. I've opened #455 to discuss it.